clusterloader/framework/util.go

```diff
@@ -80,3 +113,86 @@ func CreatePods(f *framework.Framework, appName string, ns string, labels map[st
 		}
 	}
 }
+
+func CreateRC(f *framework.Framework, namespace, name, imageName string, label map[string]string, replicas int) error {
```

Comment:
It's better to split this into multiple functions: one that creates a RC object, and one that does the creation (retries), and one that waits until it's running. Then it'll be actually possible to write unit tests for, at least some of, those functions. (Yes 'framework' doesn't follow those rules, but it doesn't mean that it's OK).